### PR TITLE
dump the namespaces at the end of e2e

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@deads2k ptal.

**Comment by deads2k** [comment]
lgtm.  You want to force merge for debugging details?

**Comment by bparees** [comment]
you are a terrible influence on me.

File: hack/test-end-to-end.sh [hunk 1]
```diff
@@ -72,6 +72,10 @@ function cleanup()
 	oc get -n docker builds --output-version=v1beta3 -t '{{ range .items }}{{.metadata.name}}{{ "\n" }}{{end}}' | xargs -r -l oc build-logs -n docker >"${LOG_DIR}/dockerbuild.log"
 	oc get -n custom builds --output-version=v1beta3 -t '{{ range .items }}{{.metadata.name}}{{ "\n" }}{{end}}' | xargs -r -l oc build-logs -n custom >"${LOG_DIR}/custombuild.log"
 
+	oc export all -n test --raw > ${LOG_DIR}/export_all_test.json
```
Comment (deads2k) [inline hack/test-end-to-end.sh]: all --all

**Pull request merged**